alpha/lib/model/UserLoginDataPeer.php

```diff
@@ -194,7 +194,7 @@ public static function checkPasswordValidation($newPassword, $loginData) {
 		
 
 	
-	public static function resetUserPassword($email)
+	public static function resetUserPassword($email, $linkType = KalturaResetPassLinkType::KMC)
```

Comment:
On the peet level, which is part of the core, use the core enum - resetPassLinkType